### PR TITLE
fix(readme): updates npm support in readme(s)

### PR DESCRIPTION
**Pull request by invincibleJai**
updates npm support in readme(s)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ Red Hat OpenShift.io services extension allows developers using more traditional
 - License analysis determines the license used by the project and its dependencies. If the project already has a license the analysis will show any conflicts between that license and dependency licenses that may be more restrictive. If the project does not have a license, OpenShift.io will suggest a license for the project that is sufficiently permissive to match the license restrictions of the dependencies.
 - Suggested dependencies that can be added to your application stack, alternative dependencies to currently used dependencies if any of the currently used dependencies are not typically used together in an application stack
 
-Today OpenShift.io services extension is limited to Vert.x and Spring Boot projects using Maven. Going forward, additional builders for Java, npm and PyPI ecosystems will be supported.
+Today OpenShift.io services extension is limited to Vert.x and Spring Boot projects using Maven and projects build on npm (Node ecosystem). Going forward, additional builders for Java, go and PyPI ecosystems will be supported.
```
Comment (sara-02) [inline README.md]: No longer limited to Vertx Sprinboot :)

File: README.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ Red Hat OpenShift.io services extension allows developers using more traditional
...
 
 ## Prerequisites
 * Maven must be installed on your machine. Provide the Maven executable filepath.
```
Comment (sara-02) [inline README.md]: We would need node, npm prerequisites as well, won't we?

**Comment by invincibleJai** [comment]
@sara-02 @samuzzal-choudhury  pls take a look.

File: README.md [hunk 3]
```diff
@@ -5,16 +5,20 @@ Red Hat OpenShift.io services extension allows developers using more traditional
...
 
  **Note:** By default, the `mvn` command is executed directly in the terminal, which requires that  `mvn` is found in your system environment `PATH`.
+ By default, the `npm` command is executed directly in the terminal, which requires that  `npm` is found in your system environment `PATH`.           
  If you do not want to add it into your system environment `PATH`, you can specify the maven executable path in settings:
+
 ```
 {
-    "maven.executable.path": "/path-to-maven-home/bin/mvn"
+    "maven.executable.path": "/path-to-maven-home/bin/mvn",
+    "npm.executable.path": "/path-to-npm-home/bin/mvn"
```
Comment (rootAvish) [inline README.md]: `"/path-to-npm-home/bin/mvn"` -> "/path-to-npm-home/bin/npm"

**Comment by invincibleJai** [comment]
@rootAvish thanks and done :)

**Pull request merged**